packages/bygger/src/translations/utils.js

```diff
@@ -68,19 +68,22 @@ const getTranslatablePropertiesFromForm = (form) =>
 const withoutDuplicatedComponents = (component, index, currentComponents) =>
   index === currentComponents.findIndex((currentComponent) => currentComponent.text === component.text);
 
-const textObject = (withInputType, value) => {
+const textObject = (withInputType, value, removeLineBreak = false) => {
   if (withInputType)
     return {
       text: value,
       type: getInputType(value),
     };
-  else
-    return {
-      text: value,
-    };
+  else {
+    return removeLineBreak
+      ? { text: value.replace(/(\r\n|\n|\r)/gm, "") }
```

Comment:
erstatt med space for å unngå at siste ord på en linje slås sammen med første ord i neste linje i csv-eksport